plasmapy/classes/tests/test_Plasma.py

```diff
@@ -111,10 +121,23 @@ def test_intermediate_and_both_energies(self):
         regimes = ['Intermediate coupling regime: Gamma = 10.585076050938532.',
                    'Both Fermi and thermal energy important: Theta = 0.04113394941084522']
 
-        assert blob.regimes() == regimes
+        testTrue = blob.regimes() == regimes
 
+        errStr = (f"Regimes \n{regimes}\n should be equal to that of class\n"
```

Comment:
I would change this to "Regime should be {regimes}, but got {blog.regimes()} instead.". That way it is clearer which is the expected answer and which is the answer from the method.